### PR TITLE
Seer

### PR DESCRIPTION
**Pull request by seerDVR**

### CODE REVIEW

File: include/fc/io/raw.hpp [hunk 1]
```diff
@@ -548,7 +548,7 @@ namespace fc {
        FC_ASSERT( _max_depth > 0 );
        --_max_depth;
        unsigned_int size; fc::raw::unpack( s, size, _max_depth );
-       FC_ASSERT( size.value*sizeof(T) < MAX_ARRAY_ALLOC_SIZE );
+       //FC_ASSERT( size.value*sizeof(T) < MAX_ARRAY_ALLOC_SIZE );
```
Comment (abitmore) [inline include/fc/io/raw.hpp:551]: Why this?

File: include/fc/container/flat_fwd.hpp [hunk 2]
```diff
@@ -3,6 +3,7 @@
 #include <boost/container/flat_set.hpp>
 #include <boost/interprocess/containers/vector.hpp>
 #include <fc/config.hpp>
+#include <stdint.h>
```
Comment (abitmore) [inline include/fc/container/flat_fwd.hpp:6]: Why? The project compiles fine in MSVC 2015 without this.

File: src/asio.cpp [hunk 3]
```diff
@@ -106,11 +106,10 @@ namespace fc {
                {
                  fc::thread::current().set_name("asio");
                  
-                 BOOST_SCOPE_EXIT(void)
+                 //BOOST_SCOPE_EXIT(void)
```
Comment (abitmore) [inline src/asio.cpp:109]: Why?

@jmjatlanta this change looks like something we discussed a few days ago. Please check.

**Review by abitmore — COMMENTED** [review]
@seerDVR please provide more info about why we need these changes.

**Comment by abitmore** [comment]
This PR looks like unintended. I think @seerDVR wanted to push to their own repository but accidentally pushed to this (upstream) repository.

**Comment by pmconrad** [comment]
Agree. Should close this in a couple of days, unless an explanation is provided.

File: src/asio.cpp (hunk 3 above)
Comment (jmjatlanta) [reply src/asio.cpp:109]: We discussed this same section of code, but it appears we were looking at solving a different problem. I believe as you do, that this is a misplaced PR, or at a minimum, needs more explanation of what issues are attempting to be solved.

**Pull request closed without merging**